content-repo/extra-docs/releases/20.9.0.md

```diff
@@ -0,0 +1,418 @@
+# Cortex XSOAR Content Release Notes for version 20.8.2 (98126)
+##### Published on 1 September 2020
+### New: EmailCommunication Pack v1.0.0
+#### IncidentFields
+- **Add CC To Email**
+#### IncidentTypes
+- **Email Communication**
+#### Scripts
+##### PreprocessEmail  
+Preprocessing script for email communication layout.
+This script checks if the incoming email contains an Incident ID to link the mail to an existing incident, and tags the email as "email-thread".
+
+For more information about the preprocessing rules, refer to: https://demisto.developers.paloaltonetworks.com/docs/incidents/incident-pre-processing
+##### SendEmailReply  
+The script sends email massages with the configured mail sender integration.
+
+---
+
+### New: Genians Pack v1.0.0 (Partner Supported)
+#### Integrations
+##### Genians  
+Use the Genians integration to block IP using assign tag and unassign tag.
+
+
+---
+
+### New: McAfee ESM Pack v1.0.0
+#### Integrations
+##### McAfee ESM v2  
+Run queries and receive alarms from Intel Security ESM. Support version 10 and above.
+
+
+---
+
+### New: Microsoft Advanced Threat Analytics Pack v1.0.0
+#### Classifiers
+#####  Microsoft Advanced Threat Analytics - Classification  
+Classifies Microsoft Advanced Threat Analytics suspicious activities
+##### Microsoft Advanced Threat Analytics  
+
+#####  Microsoft Advanced Threat Analytics - Incoming Mapper  
+Maps Microsoft Advanced Threat Analytics suspicious activity fields.
+#### IncidentFields
+- **Suspicious Activity End Time**
+- **Suspicious Activity ID**
+- **Suspicious Activity Severity**
+- **Suspicious Activity Start Time**
+- **Suspicious Activity Status**
+#### IncidentTypes
+- **Microsoft ATA Suspicious Activity**
+#### Integrations
+##### Microsoft Advanced Threat Analytics  
+Use Microsoft Advanced Threat Analytics integration to manage suspicious activities, monitoring alerts and entities.
+
+
+---
+
+## New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)
+#### Classifiers!-->
+##### NTT Cyber Threat Sensor - Classifier  
+Classifies NTT Cyber Threat Sensor incidents
+##### NTT Cyber Threat Sensor  
+
+##### NTT Cyber Threat Sensor - Incoming Mapper  
+Maps incoming NTT Cyber Threat Sensor fields.
+#### IncidentFields
+- **FAERE Description**
+- **Graph Plot**
+#### IncidentTypes
+- **TD Incident**
+#### Integrations
+##### NTT Cyber Threat Sensor  
+Retrieve alerts and recommendations from NTT CTS
+#### Playbooks
+##### Handle TD events  
+Playbook to enrich TD events
+
+
+---
+
+### New: PiHole Pack v1.0.0
```

Comment:
this is community supported
@glicht do we want to add a comment about that?

Reply:
Yes. We can add "Community Supported".